src/pages/SettingsPage/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {withRouter} from '../../libs/Router';
+import InitialPage from './InitialPage';
+import ProfilePage from './ProfilePage';
+import PreferencesPage from './PreferencesPage';
+import PasswordPage from './PasswordPage';
+import PaymentsPage from './PaymentsPage';
+
+const propTypes = {
+    match: PropTypes.shape({
+        params: PropTypes.shape({
+            // route passed via route /settings/:route
+            route: PropTypes.string,
+        }),
+    }).isRequired,
+};
+
+const subsettings = {
+    default: {
+        title: 'Settings',
+        Component: InitialPage,
+    },
+    profile: {
+        title: 'Profile',
+        Component: ProfilePage,
+    },
+    preferences: {
+        title: 'Preferences',
+        Component: PreferencesPage,
+    },
+    password: {
+        title: 'Change Password',
+        Component: PasswordPage,
+    },
+    payments: {
+        title: 'Payments',
+        Component: PaymentsPage,
+    },
+
+};
+
+const SettingsPage = ({match}) => {
+    const {route} = match.params;
```

Comment:
Do we need braces around these variables?

Reply:
it destructures the variable, basically just easier than writing `const route = match.params.route`

Reply:
oh unless you're asking if we need to destructure, just saw the style guide and sounds like for single variables we shouldn't so I'll update 👍